tests/e2e/specs/in-game/basicMoves.spec.js

```diff
@@ -57,6 +63,17 @@ describe('Game Basic Moves - P0 Perspective', () => {
       p1Points: [Card.TEN_OF_HEARTS, Card.ACE_OF_DIAMONDS],
       p1FaceCards: [Card.KING_OF_HEARTS],
     });
+
+    cy.wait(500).then(() => {
+      const consoleErrorCount = consoleError.callCount;
+
+      // Right now, we're getting a couple of:
+      // [Vue warn]: Property or method "nodeType" is not defined on the instance
+      // but referenced during render. Make sure that this property is reactive,
+      // either in the data option, or for class-based components, by initializing
+      // the property
+      cy.wrap(consoleErrorCount).should('be.lte', 8);
```

Comment:
Yeah, I agree it's wonky but wanted to try to include a test to prove that this fixes it.  Figuring out the right syntax to get the call count to be less than or equal (rather than exactly equal) was a bit more difficult than I thought it would be.

I agree the count would ideally be zero, but I didn't want to add a failing test and I didn't want this test to fail if the  other issue is resolved.

I'm not opposed to just removing this test.